challenges/objects-arrays.js

```diff
@@ -14,107 +14,157 @@
 
 // Using your dinosaur objects, log answers to these questions:
 
-// How much did tyrannosaurus weigh?
-console.log();
+// const dino1 = {
+// name: "Tyrannosaurus",
+// diet: "Carnivorous",
+// weight: "7000kg",
+// length: "1.8m",
+// period: "Late Cretaceious",
+// roar: function() {
+//   return "RAWERSRARARWERSARARARRRR!";
```

Comment:
you could also use the name as the objects name and not have to write the name key:value pair (also, note the length variable) const tyrannosaurus = { diet: "Carnivorous", weight: "7000kg", length: "12m", period: "Late Cretaceious", roar: function() { return "RAWERSRARARWERSARARARRRR!"; }